contracts/contracts/Multisig.cairo

```diff
@@ -322,80 +311,65 @@ end
 #
 
 @constructor
-func constructor{
-        syscall_ptr : felt*,
-        pedersen_ptr : HashBuiltin*,
-        range_check_ptr,
-    }(
-        owners_len : felt,
-        owners : felt*,
-        confirmations_required : felt,
-    ):
+func constructor{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}(
+    owners_len : felt, owners : felt*, confirmations_required : felt
+):
+    const lower_bound = 1
+
     with_attr error_message("invalid number of required confirmations"):
```

Comment:
As far as I can tell, this is now duplicate code. This is checked in _set_confirmation_required_impl

Reply:
The same. Just realised that all my `pending` comments can not be seen.

Reply:
Ok! Thanks for the explanation :)